scipy/stats/_hypotests.py

```diff
@@ -1630,13 +1636,26 @@ def _pval_cvm_2samp_asymptotic(t, N, nx, ny, k, *, xp):
 @_axis_nan_policy_factory(CramerVonMisesResult, n_samples=2, too_small=1,
                           result_to_tuple=_cvm_result_to_tuple)
 def cramervonmises_2samp(x, y, method='auto', *, axis=0):
-    """Perform the two-sample Cramér-von Mises test for goodness of fit.
+    r"""Perform the two-sample Cramér-von Mises test for goodness of fit.
 
     This is the two-sample version of the Cramér-von Mises test ([1]_):
     for two independent samples :math:`X_1, ..., X_n` and
     :math:`Y_1, ..., Y_m`, the null hypothesis is that the samples
     come from the same (unspecified) continuous distribution.
 
+    The statistic :math:`T` is defined as in [1]_:
+
+    .. math::
+        T = \frac{nm}{n+m}\omega^2 =
+        \frac{U}{n m (n+m)} - \frac{4 m n - 1}{6(m+n)}
+
+    where :math:`U` is defined as below. The function :math:`r()` here denotes
+    the rank of the element within the pooled sample of size :math:`n + m`, with
+    ties assigned mid-rank values:
+
+    .. math::
+        U = n \sum_{i=1}^n (r(X_i)-i)^2 + m \sum_{j=1}^m (r(Y_j)-j)^2
```

Comment:
I commented above that $x_i$ was undefined. That was assuming that $X_i$ is supposed to represent a random variable and $x_i$ is our observation of that random variable, an element in the sample. From that perspective, I'd think that $X_i$ and $Y_j$ here should be $x_i$ and $y_j$, which would also need to be defined.